app/Filament/Resources/LessonResource.php

```diff
@@ -15,6 +14,8 @@
 
 class LessonResource extends Resource
 {
+    public static string $parentResource = CourseResource::class;
```

Comment:
I think it makes more sense to put the parent resource on the child resource, instead of every child resource page

Reply:
This does make sense!